pack/fhv/fhvc.json

```diff
@@ -146,15 +146,296 @@
+    "type_code": "agenda"
+  },
+  {
+    "back_name": "By the Skin of Your Skin",
+    "back_text": "<b>If 4 or fewer damage is on The Captives:</b>\n<blockquote><i>Despite the seemingly endless onslaught, relatively few of the captives were harmed. As you survey the wrecked interior of the farmhouse, some captives weep into their hands, and others look at you accusingly. How can you convince them that their lives were worth saving?</i></blockquote>\n<b>(→R1)</b> <i>(page 50)</i>.\n<hr>\n<b>Otherwise:</b>\n<blockquote><i>The farm is a charnel house of gore and viscera. When confronted with the onslaught, many of the captives ran, screaming, into the jaws of the monstrosities. The morning light illuminates a scene that will haunt your dreams.</i></blockquote>\n<b>(→R2)</b> <i>(page 50)</i>.",
+    "clues": null,
+    "code": "10629",
+    "double_sided": true,
+    "encounter_code": "",
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "You must protect the captives!",
+    "illustrator": "Mauro Dal Bo",
+    "name": "The Longest Night",
+    "pack_code": "",
+    "position": 129,
+    "quantity": 1,
+    "stage": 1,
+    "text": "[free] During your turn, spend 1 clue: Place 1 barrier, decoy, or trap at your location.\n[action]: <b>Resign.</b>Then, if each undefeated investigator has resigned, <b>(→R3)</b> <i>(page 50).</i>\n<b>Objective</b> - Survive the night and protect as many captives as you can. <i>(Do not advance until you are instructed.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "The aspect of the whole farm was shocking— greyish withered grass and leaves on the ground, vines falling in brittle wreckage from archaic walls and gables.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
+    "back_text": "The Farmhouse cannot have player card attachments, traps, or decoys.",
+    "clues": 2,
+    "code": "10630",
+    "double_sided": true,
+    "encounter_code": "the_longest_night",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "The Farmhouse",
+    "pack_code": "fhvc",
+    "position": 130,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "The Farmhouse cannot have player card attachments, traps, or decoys.\n[free] During your turn, spend 1 [per_investigator] resources: <b>Move</b> <i>(to a connecting location).</i>\n[action]: Gain 1 clue<i> (from the token pool)</i>. (Group limit once per round.)",
+    "traits": "Sanctum.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The old farm is riddled with scraps, desiccated livestock, rotten feed, and rusted farming equipment.",
```

Comment:
There is a typo in the original card text (_dessicated_ vs _desiccated_), I took the liberty to fix it.

Reply:
I agree 100% with this fix.